airflow/models.py

```diff
@@ -4001,27 +4025,42 @@ def get_one(
 
         query = (
             session.query(cls.value)
-            .filter(and_(*filters))
-            .order_by(cls.execution_date.desc(), cls.timestamp.desc())
-            .limit(1))
+                .filter(and_(*filters))
+                .order_by(cls.execution_date.desc(), cls.timestamp.desc())
+                .limit(1))
```

Comment:
You don't need limit(1) with first (http://docs.sqlalchemy.org/en/latest/orm/query.html#sqlalchemy.orm.query.Query.first)